makelib/common.mk

```diff
@@ -250,6 +256,7 @@ build.vars:
 	@echo OUTPUT_DIR=$(OUTPUT_DIR)
 	@echo WORK_DIR=$(WORK_DIR)
 	@echo CACHE_DIR=$(CACHE_DIR)
+	@echo HELM_TOOL=$(HELM_TOOL)
 
```

Comment:
I think we should turn this into the following:
```
common.buildvars:
        @echo PROJECT_NAME=$(PROJECT_NAME)
	@echo PROJECT_REPO=$(PROJECT_REPO)
	@echo BUILD_HOST=$(HOSTNAME)
	@echo BUILD_REGISTRY=$(BUILD_REGISTRY)
	@echo DOCKER_REGISTRY=$(DOCKER_REGISTRY)
	@echo OUTPUT_DIR=$(OUTPUT_DIR)
	@echo WORK_DIR=$(WORK_DIR)
	@echo CACHE_DIR=$(CACHE_DIR)

build.vars: common.buildvars
```

and in helm.mk:

```
helm.buildvars:
     @echo HELM=$(HELM)

build.vars: helm.buildvars
```

Reply:
Sounds good! Changes implemented in following commit